or-model/src/main/java/com/google/refine/expr/MetaParser.java

```diff
@@ -70,13 +69,6 @@ static public class LanguageInfo {
 //    final static private Var CLOJURE_EVAL = RT.var("clojure.core", "eval");
     
     static {
-        registerLanguageParser("grel", "General Refine Expression Language (GREL)", new LanguageSpecificParser() {
```

Comment:
Because I moved GREL to its own module and I did not think it was worth doing that for Clojure as it's a much smaller implementation. Would you like to have Clojure in its own Maven module as well?